CHANGELOG.md

```diff
@@ -112,7 +112,7 @@ All notable changes to this project will be documented in this file. See [standa
 
 ### Reverts
 
-* **package:** use exact `uglify-es` version (`dependencies`) ([#199](https://github.com/webpack-contrib/uglifyjs-webpack-plugin/issues/199)) ([#202](https://github.com/webpack-contrib/uglifyjs-webpack-plugin/issues/202)) ([426bafd](https://github.com/webpack-contrib/uglifyjs-webpack-plugin/commit/426bafd))
+* **package:** use exact `terser` version (`dependencies`) ([#199](https://github.com/webpack-contrib/uglifyjs-webpack-plugin/issues/199)) ([#202](https://github.com/webpack-contrib/uglifyjs-webpack-plugin/issues/202)) ([426bafd](https://github.com/webpack-contrib/uglifyjs-webpack-plugin/commit/426bafd))
```

Comment:
Shouldn't change historical references of `uglify-es` versions to `terser` retroactively.

Reply:
Thanks for these! I wasn't being careful enough :/